docs/example-scenario/forensics/index.md

```diff
@@ -70,6 +70,45 @@ The Hybrid Runbook Worker must have a managed identity or a Service Principal in
 
 An Azure [Log Analytics workspace](/azure/azure-monitor/platform/resource-logs-collect-workspace) in Azure Monitor stores activity logs to audit all the events on the SOC subscription.
 
+## Regulatory Compliance
+One of the requirements when validating a CoC solution is compliance with security standards and regulations.
+All the components included in the above architecture (Azure Virtual Machine, Azure Key Vault, Azure Storage Account, Azure Log Analytics) are Azure standard services, built upon a foundation of trust, security and [compliance](https://azure.microsoft.com/overview/trusted-cloud/compliance/). Azure has more than 90 compliance certifications, including over 50 specific to global regions and countries and more than 35 specific to the needs of key industries, including health, government, finance and education.
+
+The following table shows the main services adopted in this solution and the relative standards:
```

Comment:
Instead of insert a table, is it possible to include a reference to current version? If we copy and past a subset of an external table we should keep it updated

Reply:
updated in a new version of the document